<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="PyDMPushButton" name="testButton">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Test Button</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="PyDMCheckbox" name="testCheckBox">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Test Checkbox</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="PyDMLineEdit" name="outputBox">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLineEdit</class>
   <extends>QLineEdit</extends>
   <header>pydm.widgets.line_edit</header>
  </customwidget>
  <customwidget>
   <class>PyDMPushButton</class>
   <extends>QPushButton</extends>
   <header>pydm.widgets.pushbutton</header>
  </customwidget>
  <customwidget>
   <class>PyDMCheckbox</class>
   <extends>QCheckBox</extends>
   <header>pydm.widgets.checkbox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
